smartsim/_core/_cli/build.py

```diff
@@ -170,15 +184,18 @@ def __init__(self) -> None:
                     CONFIG.conf_path = Path(CONFIG.core_path, "config", "keydb.conf")
                     if not CONFIG.conf_path.resolve().is_file():
                         raise SSConfigError(
-                            "Database configuration file at REDIS_CONF could not be found"
+                            f"Database configuration file at {CONFIG.conf_path} "
+                            + "could not be found"
```

Comment:
Per discussions with MattD, the + should be omitted in multiline string constructions